menu/startup_configuration.sh

```diff
@@ -64,7 +64,7 @@ case "$choice" in
                 run_script ADDONS locales
             fi
             input_box "Please try out all buttons (e.g: @ # \$ : y n) to find out if the keyboard settings were correctly applied.\nIf the keyboard is still wrong, you will be offered to reboot the server in the next step.\n\nPlease continue by hitting [ENTER]" >/dev/null
-            if ! yesno_box_no "Did the keyboard work as expected??\n\nIf you choose 'No' the server will be rebooted. After the reboot, please login as usual and run this script again."
+            if ! yesno_box_yes "Did the keyboard work as expected??\n\nIf you choose 'No' the server will be rebooted. After the reboot, please login as usual and run this script again."
```

Comment:
Did you forget about the `!` maybe?

Reply:
no, this is correct

Reply:
`yesno_box_yes` means, that it is by default on the `yes` button.
and the `!` means, that it will trigger the reboot, if you select `no`.